libminifi/test/TestBase.cpp

```diff
@@ -645,7 +645,7 @@ std::string TestPlan::getContent(const minifi::core::FlowFile& file) const {
   auto content_stream = content_repo_->read(*content_claim);
   auto output_stream = std::make_shared<minifi::io::BufferStream>();
   minifi::InputStreamPipe{*output_stream}(content_stream);
-  return utils::span_to<std::string>(output_stream->getBuffer().as_span<const char>());
+  return std::string{reinterpret_cast<const char*>(output_stream->getBuffer().data()), output_stream->size()};
```

Comment:
I'd prefer to keep following the as_span<const char> -> span_to<string> pattern here as well.

Reply:
done